src/api/hooks/useFormatBalance.ts

```diff
@@ -1,7 +1,7 @@
 import {Compact} from '@polkadot/types';
 import type BN from 'bn.js';
 // eslint-disable-next-line no-restricted-imports
-import {formatBalance} from '@polkadot/util';
+import {formatBalance as formatBalancePolkaUtil} from '@polkadot/util';
```

Comment:
Just to fix eslint warning about having variable with same name in the upper scope.